app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ProjectsController < ApplicationController
+	def index
+	end
+	#before_action :authenticate_user!
+	def new
+		@project = Project.new
+	end
+
+	def create
+		@project = Project.new(project_params)
```

Comment:
this could be written as : current_user.projects.new this will automatically assign user_id to project